From 7c5654c8d7839deaf4cee1d480224377f665470c Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 03 Aug 2018 13:19:36 +0300
Subject: [PATCH] Add preboot support fixup

Add build dependency with uboot, since we use its headers
---

diff --git a/package/preboot/preboot.mk b/package/preboot/preboot.mk
index 7792a9e..798d6fe 100644
--- a/package/preboot/preboot.mk
+++ b/package/preboot/preboot.mk
@@ -1,6 +1,7 @@
 THERMAL_VERSION = 1.0.0
 PREBOOT_SITE = preboot
 PREBOOT_LICENSE = unclear
+PREBOOT_DEPENDENCIES = uboot
 
 define PREBOOT_BUILD_CMDS
 	$(TARGET_MAKE_ENV) $(MAKE) CROSS_COMPILE="$(TARGET_CROSS)" -C $(@D) all
